Lib/turtle.py

```diff
@@ -2935,8 +2908,8 @@ def tiltangle(self, angle=None):
         between the orientation of the turtleshape and the heading of the
         turtle (its direction of movement).
 
-        (Incorrectly marked as deprecated since Python 3.1, it is really
-        settiltangle that is deprecated.)
+        (Incorrectly marked as deprecated since Python 3.1, it was really
+        settiltangle that was deprecated (and removed in 3.13).)
```

Comment:
I don't get the purpose of this remark. Why not just removing this part? It's misleading to have a comment about an old mistake, now fixed.

Reply:
Good point, will remove it completely now.